lib/xboxkrnl/xboxdef.h

```diff
@@ -41,6 +41,8 @@ typedef unsigned int SIZE_T, *PSIZE_T;
 typedef int BOOL, *PBOOL;
 typedef const char *PCSZ, *PCSTR, *LPCSTR;
 
+typedef unsigned __int64 QUAD;
```

Comment:
What's wrong with `LONGLONG`?
I haven't tested, but `__int64` looks to be a MSVC thing?

Reply:
Sorry about that, I had a brain fart when I was doing experimental accessing to objects. But yes, it is MSVC feature. Even weird thing that `__int128` is mentioned on [cppreference.com](https://en.cppreference.com/w/cpp/language/integer_literal) site.

Reply:
We use it already in nxdk bits. <https://github.com/XboxDev/nxdk/blob/a48ed10ae6e6e5affe090677b62d765a09cada9e/lib/winapi/winnt.h#L16>

Reply:
@RadWolfie It's an extended integer type, which GCC [doesn't support](https://gcc.gnu.org/onlinedocs/gcc/Integers-implementation.html). It seems it's not officially supported in clang either, but [prefers `_BitInt(N)` instead](https://clang.llvm.org/docs/LanguageExtensions.html#extended-integer-types).

@GXTX Intriguing. See above.